circle.yml

```diff
@@ -1,10 +1,10 @@
 machine:
-  ruby:
-    version: 2.2.4
   timezone: "America/Chicago"
```

Comment:
CircleCI will use the version from `.ruby-version`: https://circleci.com/docs/1.0/configuration/#ruby-version